README.md

```diff
@@ -157,14 +157,14 @@ struct Opts {
     subcmd: SubCommand,
 }
 
-#[derive(Clap)]
+#[derive(Parser)]
```

Comment:
This is probably unrelated to this actual change, but should this be `#[derive(Subcommand)]`? If so, we should take a harder look to see which areas we were blindly using `derive(Clap)` because it gave us everything, vs the more targeted  trait we actually needed. At least in cases where like this, we literally have a trait called `Subcommand`

Reply:
Yes, we should scrub the docs for this, similar to how we need to scrub the docs for `muiltiple_values` vs `multiple_occurrences`

Reply:
https://github.com/clap-rs/clap/issues/2815
https://github.com/clap-rs/clap/issues/2816